eclipse-collections/src/main/java/org/eclipse/collections/impl/map/AbstractMapIterable.java

```diff
@@ -55,6 +55,12 @@ protected boolean isAbsent(V result, K key)
         return result == null && !this.containsKey(key);
     }
 
+    @Override
```

Comment:
Is this necessary?

Reply:
Yes for the Immutable side

Reply:
For transparency the default methods and implementations other than MapIterable are to fix compiler complaints, and I have put them at the highest place.

Reply:
Not sure I understand. `ImmutableMap` extends `ImmutableMapIterable` which extends `MapIterable` so all `ImmutableMap` implementations should get the `default` implementation and the override in `AbstractMapIterable` should be unnecessary right?

Reply:
@donraab It is necessary because `AbstractImmutableMap implements ImmutableMap, Map`. So the immutable side gets the ambiguous method issue. Screenshot below:
![image](https://user-images.githubusercontent.com/9932381/88483834-225b2580-cf30-11ea-93ee-a25c45e62e11.png)